<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Option</class>
 <widget class="QDialog" name="Option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>메인 화면 웹 설정 (재시작 후 적용)</string>
     </property>
     <widget class="QTextEdit" name="URL">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>382</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QCheckBox" name="URL_Check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>사용자 지정 URL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="URL_reset">
      <property name="geometry">
       <rect>
        <x>311</x>
        <y>75</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 80, 0);</string>
      </property>
      <property name="text">
       <string>URL 초기화</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>RB Editor 미리 불러오기</string>
     </property>
     <widget class="QTextEdit" name="Editor">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>382</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>파일 주소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Editor_Btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>74</y>
        <width>96</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 186, 186);</string>
      </property>
      <property name="text">
       <string>불러오기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Editor_reset">
      <property name="geometry">
       <rect>
        <x>311</x>
        <y>75</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 80, 0);</string>
      </property>
      <property name="text">
       <string>주소 초기화</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="Option_Btn">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>RB Links 바로가기 설정</string>
     </property>
     <widget class="QTextEdit" name="Link">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>382</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>바로가기 주소</string>
      </property>
     </widget>
     <widget class="QPushButton" name="File_Btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>74</y>
        <width>96</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 186, 186);</string>
      </property>
      <property name="text">
       <string>파일 불러오기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Folder_Btn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>74</y>
        <width>96</width>
        <height>26</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 186, 186);</string>
      </property>
      <property name="text">
       <string>폴더 불러오기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Link_reset">
      <property name="geometry">
       <rect>
        <x>311</x>
        <y>75</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 80, 0);</string>
      </property>
      <property name="text">
       <string>주소 초기화</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Option_Btn</sender>
   <signal>accepted()</signal>
   <receiver>Option</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Option_Btn</sender>
   <signal>rejected()</signal>
   <receiver>Option</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>URL_Check</sender>
   <signal>clicked(bool)</signal>
   <receiver>URL</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
